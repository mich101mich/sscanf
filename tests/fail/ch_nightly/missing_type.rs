fn main() {
    sscanf::scanf!("hi", "asdf{}{}asdf{:bob}", usize);
}
