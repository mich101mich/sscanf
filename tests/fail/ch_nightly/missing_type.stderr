error: Missing Type for given '{}' Placeholder
 --> $DIR/missing_type.rs:2:33
  |
2 |     sscanf::scanf!("hi", "asdf{}{}asdf{}", usize);
  |                                 ^^

error: Missing Type for given '{}' Placeholder
 --> $DIR/missing_type.rs:2:39
  |
2 |     sscanf::scanf!("hi", "asdf{}{}asdf{}", usize);
  |                                       ^^

error: Missing Type for given '{}' Placeholder
 --> $DIR/missing_type.rs:3:42
  |
3 |     sscanf::scanf!("hi", "asdf{}{usize}as{}df{i32}", usize);
  |                                          ^^
